<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1327</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="InventoryTW">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>621</width>
      <height>651</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>SKU</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Quantity</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Category</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="Grocery">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>300</y>
      <width>451</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Grocery</string>
    </property>
    <widget class="QLabel" name="CategoryLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Category:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="CategoryCB">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>251</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Beverages</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dairy Products</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fruit and vegetables</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bakery</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QComboBox" name="OptionCB">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>20</y>
      <width>281</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Grocery</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Clothing</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="Clothing">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>280</y>
      <width>451</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Clothing</string>
    </property>
    <widget class="QComboBox" name="SizeCB">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>XS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>S</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>M</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>L</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>XL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>XXL</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="ColorCB">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Black</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Red</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Blue</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>White</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Green</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="MaterialCB">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Wool</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cotton</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lawn</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Linen</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="SizeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
    <widget class="QLabel" name="ColorLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
    <widget class="QLabel" name="MaterialLable">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>140</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Material</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="InventoryTW_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>851</width>
      <height>651</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>SKU</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Quantity</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Size</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Color</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Material</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>500</y>
      <width>271</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="RemovePB">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="AddPB">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="LoadPB">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="SavePB">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>60</y>
      <width>381</width>
      <height>185</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="SKULabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>SKU:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="SKULE">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Namelabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="NameLE">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="QuantityLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Quantity</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="PriceLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Price</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="PriceLineEdit">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="QuantityLE">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>Grocery</zorder>
   <zorder>InventoryTW</zorder>
   <zorder>PriceLineEdit</zorder>
   <zorder>QuantityLE</zorder>
   <zorder>NameLE</zorder>
   <zorder>Namelabel</zorder>
   <zorder>QuantityLabel</zorder>
   <zorder>PriceLabel</zorder>
   <zorder>SKULabel</zorder>
   <zorder>SKULE</zorder>
   <zorder>OptionCB</zorder>
   <zorder>Clothing</zorder>
   <zorder>AddPB</zorder>
   <zorder>RemovePB</zorder>
   <zorder>LoadPB</zorder>
   <zorder>SavePB</zorder>
   <zorder>InventoryTW_2</zorder>
   <zorder>SalePB</zorder>
   <zorder>spinBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1327</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>SKULE</tabstop>
  <tabstop>NameLE</tabstop>
  <tabstop>QuantityLE</tabstop>
  <tabstop>PriceLineEdit</tabstop>
  <tabstop>AddPB</tabstop>
  <tabstop>RemovePB</tabstop>
  <tabstop>LoadPB</tabstop>
  <tabstop>OptionCB</tabstop>
  <tabstop>SavePB</tabstop>
  <tabstop>SizeCB</tabstop>
  <tabstop>ColorCB</tabstop>
  <tabstop>MaterialCB</tabstop>
  <tabstop>InventoryTW_2</tabstop>
  <tabstop>CategoryCB</tabstop>
  <tabstop>InventoryTW</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
